docs/manuals/scripts/generate-supported-versions-timeline.py

```
@@ -0,0 +1,486 @@
+        row_center = row_y + BAR_HEIGHT / 2
+        next_release = lookup.get(release.major + 1)
+
+        current_end = next_release.release_date if next_release else window_end
+        support_end = support_end_date(
+            release, lookup, supported_major_releases, window_end
+        )
+
+        label = (
+            f"Bareos {release.major}"
+            if release.projected
+            else f"Bareos {release.major}"
```

Ruff (0.15.15)

[warning] Useless `if`-`else` condition

(RUF034)